config.go

```diff
@@ -58,10 +64,9 @@ func NewDemoConfig() *Config {
 
 }
 
-// NewConfig initiates the config with default values.
-func NewConfig(uuid string) *Config {
+func NewConfigWithUserId(userId UserId) *Config {
 	c := Config{
-		UUID:                          uuid,
```

Comment:
Is there a reason why we don't rename `UUID` to `UserId` within the Config?

Reply:
Yes, indeed. The reason is that we don't want to break the code, yet. We might need to give a warning at least that sooner or later `UUID` will be removed.